libs/platform/src/lib/components/Forms/radio-group/radio-group.component.html

```diff
@@ -0,0 +1,31 @@
+<fd-form-group [isInline]="isInline">
+    <div fd-form-item *ngIf="hasNoValue">
+        <fd-radio-button
+            fd-form-control
+            name="name"
+            [value]="None"
```

Comment:
Is "None" a string value? If so, this will not work unless the string is placed in single quotes, like this:
```
[value]="'None'"
```

Reply:
you dont use to use it this way. `[value]="None"` must even fail the compilation no?  you are using binding syntax. 
Also whey are u using directive `fd-form-control` ?